packages/rollup-plugin-lit-css/README.md

```diff
@@ -22,13 +23,13 @@ In the mean time, enjoy importing your CSS into your component files.
 
 ## Options
 
-|Name|Accepts|Default|
-|-----|-----|-----|
-|`include`|Array of glob of files to include.|`['**/*.css']`|
-|`exclude`|Array of glob of files to exclude. |`undefined`|
-|`uglify`|Boolean or Object of [uglifycss](https://www.npmjs.com/package/uglifycss#api) options.|`false`|
-|`specifier`|Package to import `css` from|`lit`|
-|`tag`|Name of the template-tag function|`css`|
+| Name        | Accepts                                                                                | Default        |
```

Comment:
Shouldn't `transform` be documented here as well?

Reply:
Thanks!